.github/workflows/go.yaml

```diff
@@ -48,10 +48,10 @@ jobs:
           go-version-file: 'go.mod'
           cache: false
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v6
+        uses: golangci/golangci-lint-action@v7
```

Comment:
Can you please pin the version to the hash of the latest v7 release and add the comment referencing that release?